### ISSUE TITLE
Investigate why the feedback e-mails do not come to the set mail.

### ISSUE THREAD

**Issue by abaranowska1** [benetech/ServiceNet#1752]
**Bug description**
According to #1133 after sending a feedback email it should be delivered. During regression, I asked @johnhbenetech and he confirmed that he didn't get an email. Please investigate why. 

**To Reproduce**
1. Go to the Feedback page in the header
2. Send a message. 

Dev Signoffs
- [x] Unit test added for the core business features at the minimum
- [x] Code review passed
- [x] CI passed successfully
- [x] CodeQL scan for vulnerabilities in code has passed 
- [x] Docker images have been built and deployed to the dev server

QA Signoffs
- [x] Passes Acceptance Criteria
- [x] All bug reports fixed/merged/tested
- [ ] Regression test cases cycle updated if needed

Product Signoff
- [x] Approved for release

**Comment by jkondrat** [comment]
@johnhbenetech it seems that it's no longer allowed to set 'from' address when sending out emails with Sendgrid (it still returns code 202, the error message can probably be found in the Sengrid app). I will try setting 'Reply-to' header, if that doesn't work perhaps we can include something like 'Please reply to user@thatsentfeedback.net' in the body of the message?

**Comment by jkondrat** [comment]
@johnhbenetech as per docs - the sender identity has to be verified https://sendgrid.com/docs/for-developers/sending-email/sender-identity/

**Comment by johnhbenetech** [comment]
@jkondrat 

Yes, let's just include it in the body, The from address can be 'feedback@benetech.org'

**Comment by jkondrat** [comment]
@johnhbenetech I was able to use reply-to header, is this sufficient? the delivered email looks like this:
![Screenshot from 2021-02-23 18-55-38](https://user-images.githubusercontent.com/5969801/108886054-cf9e3500-7608-11eb-8ab5-1d4d6d4bacc8.png)
![Screenshot from 2021-02-23 18-54-31](https://user-images.githubusercontent.com/5969801/108886159-ec3a6d00-7608-11eb-8832-ee5ea2eee87d.png)

**Comment by johnhbenetech** [comment]
@jkondrat looks good

**Comment by abaranowska1** [comment]
Feedback emails are sent properly once again.
